web_server/authors/views.py

```diff
@@ -202,25 +206,29 @@ def post_creation_and_retrival_to_curr_auth_user(request):
         new_post = Post()
 
         # new_post.id = post['id']                  #: "de305d54-75b4-431b-adb2-eb6b9e546013",
-        new_post.title       = post['title']        #: "A post title about a post about web dev",
+        #: "A post title about a post about web dev",
+        new_post.title = post['title']
         # new_post.source      = post['source']       #: "http://lastplaceigotthisfrom.com/posts/yyyyy"
         # new_post.origin      = post['origin']       #: "http://whereitcamefrom.com/posts/zzzzz"
-        new_post.description = post['description']  # : "This post discusses stuff -- brief",
+        # : "This post discusses stuff -- brief",
+        new_post.description = post['description']
         # new_post.contentType = post['contentType']  # : "text/plain",
-        new_post.content     = post['content']      #: "stuffs",
-        new_post.author      = request.user         # the authenticated user
+        new_post.content = post['content']      #: "stuffs",
+        new_post.author = request.user         # the authenticated user
         # Categories added after new post is saved
-        new_post.count       = 0                    #: 1023, initially the number of comments is zero
-        new_post.size        = size                 #: 50,
+        #: 1023, initially the number of comments is zero
+        new_post.count = 0
+        new_post.size = size                 #: 50,
```

Comment:
This looks like you've been using automatic code style fixing tools on the entire file whenever you make changes. Would you mind only doing this for code you change in the future? The style you have isn't always the most readable style and adds lots of changes to your commits that make it hard to tell what is going on.